tools/hicup/hicup_filter.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="hicup_filter" name="Filter" version="0.1.0">
+<tool id="hicup_filter" name="Hicup_Filter" version="0.1.0">
```

Comment:
This was not needed as the ID will be picked up as repository name.
If you want to include Hicup in the name, please use a whitespace. This is directly exposed to the user.
Btw. why is the "description" tag missing in the tools.